.github/actions/checkDeployBlockers/checkDeployBlockers.js

```diff
@@ -0,0 +1,46 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const {GITHUB_OWNER, EXPENSIFY_CASH_REPO} = require('../../libs/GithubUtils');
+
+const run = function () {
+    const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+    const issueNumber = Number(core.getInput('ISSUE_NUMBER', {required: true}));
+    let hasDeployBlockers = false;
+
+    return octokit.rest.issues.get({
+        owner: GITHUB_OWNER,
+        repo: EXPENSIFY_CASH_REPO,
+        issue_number: issueNumber,
+    })
+        .then((issue) => {
+            const body = issue.data.body || '';
+            const pattern = /-\s\[\s]/g;
+            const matches = pattern.exec(body);
+            hasDeployBlockers = matches !== null;
+
+            return issue.data;
+        })
+        .then(() => {
+            if (!hasDeployBlockers) {
+                return octokit.rest.issues.listComments({
+                    owner: GITHUB_OWNER,
+                    repo: EXPENSIFY_CASH_REPO,
+                    issue_number: issueNumber,
+                })
+                    .then((comments) => {
+                        const lastComment = comments.data[comments.data.length - 1];
```

Comment:
Interesting thought... what if:

1) We had an issue with all the boxes checked but no :shipit:
2) It is closed, fails this action, and then `OSBotify` comments saying "This issue either has unchecked QA steps or has not yet been marked with the `:shipit:` emoji"
3) The issue is closed again, but this time this action passes because a comment made by `OSBotify` includes `:shipit:`

Is that covered by some cleverness of this regex, or should we filter the list of comments to remove any made by `OSBotify`?

Reply:
Yep! The `^` in `/^:shipit:/g` says "Look for the string `:shipit:` at the start of a new line"